src/App.js

```diff
@@ -1,28 +1,24 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import { Header } from "./component/Header";
+import { Home } from "./component/Home";
+import { Restaurants } from "./component/Restaurants";
+import { OrderHistory } from "./component/OrderHistory";
 
-class App extends Component {
-  render() {
-    return (
+
+function App() {
+  return (
+    <BrowserRouter>
       <div className="App">
```

Comment:
Is this classname being used by the PMO styles, or was this just leftover from the original?